float64[] ik_solution

